### PR TITLE
Move Restart to Overflow and yaml config advanced

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/core/ha-config-system-navigation.ts [hunk 1]
```diff
@@ -60,6 +79,23 @@ class HaConfigSystemNavigation extends LitElement {
...
 
+  private _handleAction(ev: CustomEvent<ActionDetail>) {
+    switch (ev.detail.index) {
+      case 0:
+        showConfirmationDialog(this, {
+          text: this.hass.localize(
+            "ui.panel.config.server_control.section.server_management.confirm_restart"
+          ),
+          confirmText: this.hass!.localize("ui.common.leave"),
+          dismissText: this.hass!.localize("ui.common.stay"),
```
Comment (balloob) [inline src/panels/config/core/ha-config-system-navigation.ts:90]: Are these not the default for the confirmation dialog?

File: src/panels/developer-tools/ha-panel-developer-tools.ts [hunk 2]
```diff
@@ -67,6 +67,9 @@ class PanelDeveloperTools extends LitElement {
                 "ui.panel.developer-tools.tabs.statistics.title"
               )}
             </paper-tab>
+            <paper-tab page-name="yaml">
```
Comment (balloob) [inline src/panels/developer-tools/ha-panel-developer-tools.ts:70]: Should it be the first ?

**Review by balloob — DISMISSED** [review]
We should rename the keys in lokalise before merging this PR since they have the same content.

File: src/panels/developer-tools/ha-panel-developer-tools.ts (hunk 2 above)
Comment (zsarnett) [reply src/panels/developer-tools/ha-panel-developer-tools.ts:70]: :shrug:

I feel like if we change the first tab people riot. If we don't put it first people riot. Do we just wait and see?

**Comment by balloob** [comment]
Translations renamed.

**Pull request merged**